src/App.vue

```diff
@@ -66,9 +69,15 @@ export default {
       errored: false,
       errors: [],
       beerSearch: '',
-      beerStrength: 'all'
+      beerStrength: 'all',
+      showFavourites: false
     };
   },
+  methods: {
+    toggleFavourites() {
+      return (this.showFavourites = !this.showFavourites);
```

Comment:
This is a bit weird. Why are you return the result of an assignment? What does it return exactly and where is it used?

Reply:
This method is toggling the state of the data object property showFavourites.

This is then used in the filter

```
        .filter(beer => {
          if (this.showFavourites) {
            return beer.isFavourite;
          }

          return beer;
        });
```

And use in the computed property for the button text

```
    showFavouriteText() {
      return this.showFavourites ? 'Show all beers' : 'Show favourites';
    }
```